packages/clangml/clangml.3.5.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+authors: ["Pippijn van Steenhoven"]
+maintainer: "francois.berenger@inria.fr"
+homepage: "https://github.com/Antique-team/clangml"
+bug-reports: "https://github.com/Antique-team/clangml/issues"
+dev-repo: "https://github.com/Antique-team/clangml.git"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "batteries"
+  "deriving"
+  "ANSITerminal"
+  "base-unix"
+  "camlp4"
+  "ocamlfind"
+  "ocamlbuild" {build}
+]
+depexts: [
+  [["debian"] ["libboost-dev" "llvm-3.5-dev" "clang-3.5" "libclang-3.5-dev"
+               "binutils-dev"]]
+  [["ubuntu"] ["libboost-dev" "llvm-3.5-dev" "clang-3.5" "libclang-3.5-dev"
+               "binutils-dev"]]
+  [["gentoo"] ["dev-libs/boost" "sys-devel/llvm-3.5.0"
+               "sys-devel/clang-3.5.0-r100" "sys-devel/binutils"]]
+  [["archlinux"] ["boost" "binutils"]]
+]
+available: [
+  ocaml-version >= "4.02.3" & !preinstalled
+]
```

Comment:
Per Antique-team/clangml#56, this doesn't build on OS X. Could you please add `os != "darwin"` here?